test/unit/trigger-method.spec.js

```diff
@@ -160,12 +160,12 @@ describe('trigger event and method name', function() {
 
     describe('when the context does not have triggerMethod defined', function() {
       beforeEach(function() {
-        this.view = new Backbone.View();
-        this.view.onFoo = this.methodHandler;
-        this.view.on('foo', this.eventHandler);
+        this.obj = _.extend({}, Backbone.Events);
+        this.obj.onFoo = this.methodHandler;
+        this.obj.on('foo', this.eventHandler);
         this.triggerMethodSpy = this.sinon.spy(Marionette, 'triggerMethod');
 
-        Marionette.triggerMethodOn(this.view, 'foo');
+        Marionette.triggerMethodOn(this.obj, 'foo');
```

Comment:
This is some great TJK code that uses B.Events directly because he was re-mixing in the B.Events after he classified it. I kept this in because well, when I go back i don't want to miss it :)